isthmus/src/main/java/io/substrait/isthmus/SqlConverterBase.java

```diff
@@ -217,9 +210,10 @@ public DefinedTable(String name, RelDataTypeFactory factory, RelDataType type) {
 
     @Override
     public RelDataType getRowType(RelDataTypeFactory typeFactory) {
-      //      if (factory != typeFactory) {
-      //        throw new IllegalStateException("Different type factory than previously used.");
-      //      }
+      // if (factory != typeFactory) {
+      // throw new IllegalStateException("Different type factory than previously
+      // used.");
+      // }
```

Comment:
not sure what policy is for `substrait-java` but if you are anyway touching those lines I guess they can also be removed

Reply:
Yes; though they were there before this PR, so not sure why they shown as changed. or why they were commented out at all. 

Might as well remove them - they will be in the git history if needed